docs/loading/RoutineLoad.md

```diff
@@ -10,7 +10,7 @@ Routine Load supports data transformation at data loading and supports data chan
 
 ## Supported data formats
 
-Routine Load now supports consuming CSV and JSON format data from a Kafka cluster.
+Routine Load now supports consuming CSV, JSON and Avro (supported since v3.0.1) format data from a Kafka cluster.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/303b0cdb-ea3d-432f-9f84-fd7cb49300bd)
1) add a comma (,) preceding "and"?
2) change "format data" to "formatted data"?